envi/archs/arm/disasm.py

```diff
@@ -2099,11 +2150,11 @@ def getOperAddr(self, op, emu=None):
 
         
         if (self.pubwl & 0x12) == 0x12:    # pre-indexed
-            if emu._forrealz: emu.setRegister( self.base_reg, addr)
+            if (emu != None) and (emu._forrealz): emu.setRegister( self.base_reg, addr)
```

Comment:
Not sure I understand this comment. See Literal. Literal literally is the same as immediate except using PC as register. Difference is that when the reg is PC the calculation is done by the assembler at time of compilation so the indexing is still done, just at compile time so the resultant number displayed is with the index applied instead of displaying the index.